nvflare/client/in_process/api.py

```diff
@@ -234,3 +234,7 @@ def __continue_job(self) -> bool:
             return False
 
         return True
+
+    def shutdown(self):
+        self.stop = True
+        self.stop_reason = "API shutdown called."
```

Comment:
Any resources to clean up? Anything to stop?

Reply:
added "self.event_manager.fire_event(TOPIC_STOP)"